app/assets/stylesheets/components/_reveal.scss

```diff
@@ -0,0 +1,29 @@
+.reveal {
+  color: #034E46;
+  border: 2px solid #034E46;
+  background-color: #EBFFFA;
+  border-radius: 10px;
+  padding: .2em .5em .3em .7em;
```

Comment:
these are copied from `.reveal__link` in honeycrisp - the design had the green box around the whole thing so i sort of just moved the styles from the child element to the parent